decoder/drc_src/impd_drc_api.c

```diff
@@ -246,7 +246,7 @@ IA_ERRORCODE ia_drc_dec_api(pVOID p_ia_drc_dec_obj, WORD32 i_cmd, WORD32 i_idx,
     case IA_API_CMD_SET_CONFIG_PARAM: {
       switch (i_idx) {
         case IA_DRC_DEC_CONFIG_PARAM_SAMP_FREQ: {
-          if (*pus_value < 8000 || *pus_value > 96000) {
+          if (*pus_value <= 0 || *pus_value > 96000) {
```

Comment:
We are doing comparison of unsigned integer. So, should the check be `*pus_value == 0 || *pus_value > 96000` rather ?